tiledbcontents/tiledbcontents.py

```diff
@@ -415,6 +479,15 @@ def _create_array(self, uri, retry=0):
                 ctx=tiledb_create_context,
             )
 
+            parts = uri.split("/")
```

Comment:
I think `parts`, `parts_len` and `namespace` are also computed above at line 423?